src/design-tokens/tier-1-definitions/fonts.css

```diff
@@ -0,0 +1,17 @@
+@font-face {
+  font-family: 'Graphik';
+  font-weight: 400;
+  src: url('./fonts/Graphik-Regular-Web.woff2') format('woff2');
```

Comment:
According to [caniuse](https://caniuse.com/?search=woff2), all our supported browsers should support `'woff2'`. we could add `'woff'` lines if we want to be extra safe